aldryn_reversion/tests/test_forms.py

```diff
@@ -0,0 +1,214 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import reversion
+from cms.models import Placeholder
+
+from aldryn_reversion.test_helpers.test_app.models import (
+    SimpleNoAdmin, SimpleFK, WithPlaceholder, WithTranslations,
+)
+
+from ..forms import RecoverObjectWithTranslationForm
+from ..utils import (
+    resolve_conflicts, get_deleted_placeholders_for_object,
+)
+
+from .base import (
+    ReversionBaseTestCase, HelperModelsObjectsSetupMixin,
+    get_latest_version_for_object
+)
+
+
+class FormsTestCase(HelperModelsObjectsSetupMixin, ReversionBaseTestCase):
+    form_class = RecoverObjectWithTranslationForm
+
+    def build_form_kwargs(self, obj):
+        version = get_latest_version_for_object(obj)
+        return {
+            'revision': version.revision,
+            'obj': obj,
+            'version': version,
+            'resolve_conflicts': [],
+            'placeholders': [],
+        }
+
+    def build_unbound_form(self, arguments=None, **kwargs):
+        return self.form_class(
+            **(arguments if arguments is not None else kwargs))
```

Comment:
Well it expects arguments to be a dict, and if it is a dict then why not?
At least when i've checked that quickly it was looking like working.